psfr/psfr.py

```diff
@@ -419,9 +471,16 @@ def _minimize(x, data, model, variance=None, mask=None, oversampling=1):
         return chi2
 
     init = np.array([0, 0])
-    x = scipy.optimize.minimize(_minimize, init, args=(data, model, variance, mask, oversampling),
-                                bounds=((-5, 5), (-5, 5)), method='Nelder-Mead')
-    return x.x
+
+
+    if optimizer_type == 'NM':
+        x = scipy.optimize.minimize(_minimize, init, args=(data, model, variance, mask, oversampling),
+                                bounds=((-1.5, 1.5), (-1.5, 1.5)), method='Nelder-Mead')
+        return x.x
+    if optimizer_type =='emcee':
+        x =_minimize_emcee(init,data,model,variance, mask,oversampling, nwalkers = 10,niter= 100,startStep = 0.1)
+
```

Comment:
add 'else:' statement and then give a raise statement:
else:
 raise ValueError('optimization type %s is not supported. Please use xxx instead.' % optimizer_type)